packages/firestore/src/api/settings.ts

```diff
@@ -81,6 +81,8 @@ export interface FirestoreSettings extends LiteSettings {
    * may be removed in a future release. If you find yourself using it to
    * work around a specific network reliability issue, please tell us about
    * it in https://github.com/firebase/firebase-js-sdk/issues/1674.
+   *
+   * This setting cannot be used in a Node.js environment.
```

Comment:
Of the three slightly different phrasings, I like this best -- or maybe a combo, like "This x does not work in a Node.js environment."

Reply:
Changed all to "does not work in a Node.js environment."